tests/Feature/Api/PublicScheduleTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Tests\Feature\Api;
+
+use App\Models\Lecturing;
+use Carbon\Carbon;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+
+class PublicScheduleTest extends TestCase
+{
+    use RefreshDatabase;
+
+    /** @test */
+    public function can_get_schedules_within_specified_date_range() {
+        $lecturing = factory(Lecturing::class)->create([
+            'audience_code' => Lecturing::AUDIENCE_FRIDAY,
+            'date' => Carbon::tomorrow()->format('Y-m-d')
+        ]);
+
```

Comment:
Kita pastikan pengajian diluar rentang tanggal tidak kelihatan.

```suggestion
        $unlistedLecturing = factory(Lecturing::class)->create([
            'audience_code' => Lecturing::AUDIENCE_PUBLIC,
            'date' => Carbon::yesterday()->format('Y-m-d'),
        ]);
```